tests/interfaces/test_jax_qnode.py

```diff
@@ -827,21 +859,21 @@ def cost_fn(a, b):
         assert spy.call_args[1]["gradient_fn"] is qml.gradients.param_shift
 
 
-@pytest.mark.parametrize("dev_name,diff_method,mode", qubit_device_and_diff_method)
+@pytest.mark.parametrize("dev_name,diff_method,grad_on_execution", qubit_device_and_diff_method)
 class TestQubitIntegration:
     """Tests that ensure various qubit circuits integrate correctly"""
 
-    def test_sampling(self, dev_name, diff_method, mode):
+    def test_sampling(self, dev_name, diff_method, grad_on_execution):
         """Test sampling works as expected"""
-        if mode == "forward":
-            pytest.skip("Sampling not possible with forward mode differentiation.")
+        if grad_on_execution:
```

Comment:
Could this be "best"? And if so, should this test be skipped?

Reply:
We don't use best in the test.

Reply:
I updated it in the new tests wiht `is True`